packages/browser-wallet/src/popup/pages/IdProofRequest/DisplayStatement/DisplayStatement.tsx

```diff
@@ -0,0 +1,266 @@
+/* eslint-disable react/prop-types */
```

Comment:
![Screenshot from 2022-12-07 12-46-04](https://user-images.githubusercontent.com/87635671/206171549-af442ae4-e377-49a0-bea5-654b6e761b01.png)
:thinking:

Reply:
For:
Attribute in range
dob
Lower: 19000505
Upper: 20000505

Reply:
Fixed, sorry!